# Copyright 2022 Tiernan8r
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
UI_FILENAME = "form.ui"
GRAPH_WIDGET_NAME = "graph"

INPUT_SEARCH_WIDGET_NAME = "input_search"
INPUT_TARGET_WIDGET_NAME = "input_target"
INPUT_BROWSE_FILE = "browse_input_button"

BUTTON_SEARCH_BUTTON = "search_button"
BUTTON_CANCEL_SEARCH_BUTTON = "cancel_button"
BUTTON_PROGRESS_BAR = "progress_bar"
BUTTON_PROGRESS_BAR_TICK_RATE = 100  # in milliseconds

LCD_CLASSICAL = "lcd_number_classic"
LCD_GROVER = "lcd_number_grover"

THREAD_PAUSE = 0.01
